apps/ws/src/socket/usecases/external-services-route/external-services-route.usecase.ts

```diff
@@ -12,42 +11,41 @@ const LOG_CONTEXT = 'ExternalServicesRoute';
 
 @Injectable()
 export class ExternalServicesRoute {
-  public readonly bullMqService: BullMqService;
+  constructor(private wsGateway: WSGateway, private messageRepository: MessageRepository) {}
 
-  constructor(private wsGateway: WSGateway, private messageRepository: MessageRepository) {
-    this.bullMqService = new BullMqService();
```

Comment:
This wasn't used. The dependency with BullMQ is attached in the WebSocketWorker in this same app.